interfaces/builtin/opengl.go

```diff
@@ -113,12 +113,14 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 /run/udev/data/c226:[0-9]* r,  # 226 drm
 `
 
-// The nvidia modules don't use sysfs (therefore they can't be udev tagged) and
+// Some nvidia modules don't use sysfs (therefore they can't be udev tagged) and
 // will be added by snap-confine.
 var openglConnectedPlugUDev = []string{
 	`SUBSYSTEM=="drm", KERNEL=="card[0-9]*"`,
 	`KERNEL=="vchiq"`,
 	`KERNEL=="renderD[0-9]*"`,
+	`KERNEL=="nvhost-*"`,
+	`KERNEL=="nvmap"`,
```

Comment:
Unless those are GPL and `udev` sees them you need to also change snap-confine's `udev-support.c`

Reply:
I'm not sure if they are GPL or not, but `udev` does see them.

Reply:
Are they represented in sysfs?

Reply:
Yes: https://paste.ubuntu.com/p/GQVRy9ychP/ (see my previous comment https://github.com/snapcore/snapd/pull/6623#discussion_r267419812)